serve/graph/model/transaction.go

```diff
@@ -4,41 +4,133 @@ import (
 	"encoding/base64"
 	"fmt"
 
+	"github.com/gnolang/gno/gno.land/pkg/sdk/vm"
+	"github.com/gnolang/gno/tm2/pkg/amino"
 	"github.com/gnolang/gno/tm2/pkg/bft/types"
+	"github.com/gnolang/gno/tm2/pkg/sdk/bank"
+	"github.com/gnolang/gno/tm2/pkg/std"
 )
 
 type Transaction struct {
-	t *types.TxResult
```

Comment:
Why is this change needed?

Reply:
This doesn't need to be changed, 
but I thought there might be some confusion when implementing the function, so I made a small modification.